### DISCUSSION TITLE
Extended Sorting/Filtering Functionality for Saved Articles

### DISCUSSION THREAD

**Discussion by CarrollDiehl** [spacecowboy/Feeder#178] [Ideas]
Hi!

First of all, thank you for the great app. I'm using and enjoying Feeder daily for around a year now.

But there is one thing, where I would wish for extended functionallity: **Finding read articles**:

I use Feeder mainly for news and journalism, so I built up a large list of `Saved Articles` so far. Unfortunately, this is really just an unsorted list, which I can't order or search through.

Useful features (in order of usefulness) I'd consider for saved articles:

1. A search function to filter the feeds by title (and maybe, optionally also for fetched preview). This would not only be useful within the saved articles, but within the feeds themselves.
2. The possibility to tag feeds. For example 'read list', 'reportage', 'opinions', 'people', 'politics' etc. This would also require the functionality to filter for these tags (favourably for multiple tags).
3. Filter by feed, the same way as in the feeds itself.
4. Not connected to ordering: The ability to save the full articles. Sometimes sources go offline or I do. I couldn't find a feature yet, which allows me to save the full article, so I can re-read it without internet connection.

When I am looking for recently read articles, which I didn't save but maybe want to refer to or show a friend, I often have to scroll a lot until I find it. I read around 25% of the feeds I receive, so most articles in my feeds are actually unread.
Unfortunately, the `Unread` filter can't be unchecked. I think it might be a good idea to enable it. It could be reset on app start, so users don't 'break' their feeds.

Thank you, for developing this app!

A good start into the year
Carroll

**Comment by DraconicNEO** [comment]
I very much agree. Especially when it comes to the idea of the search function. I'm not sure why we don't have the ability to search for articles by title or content already within the app, this would be such a useful feature.

**Reply by spacecowboy** [reply]
Because no one has built it. Feeder is free software and contributions are welcome.
